### PR TITLE
documentation updates

### PR DESCRIPTION
**Pull request by tdonaworth**
## What changed

Minor updates to README.md and 012-ADR

## Issue

PRs were committed without applying suggested changes.

### CODE REVIEW

**Review by halprin — CHANGES_REQUESTED** [review]
You also need to rename the title of this ADR.  I.e. the first line in this document.

**Comment by tdonaworth** [comment]
> You also need to rename the title of this ADR. I.e. the first line in this document.

Fixed the 012 and 011 - since the GitHub issue was still numbered #010 as well

File: docs/adr/011-github-actions-for-ci-cd.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# 10. Use GitHub Actions for CI/CD
+# 11. Use GitHub Actions for CI/CD
```
Comment (halprin) [inline docs/adr/011-github-actions-for-ci-cd.md:1]: Thanks for this!  Good catch.

**Review by halprin — APPROVED** [review]
🎸 Rock and roll!

**Pull request merged**